src/main/java/org/forgerock/openam/examples/SampleAuth.java

```diff
@@ -42,67 +42,72 @@
 import com.sun.identity.shared.datastruct.CollectionHelper;
 import com.sun.identity.shared.debug.Debug;
 
-
-
-public class SampleAuth extends AMLoginModule
-{
+/**
+ * SampleAuth authentication module example.
+ *
+ * If you want to create your own module based on this example, you will need to modify every "SampleAuth" occurrences
```

Comment:
If you create your own module based on this example, you must modify all occurrences of "SampleAuth" in addition to changing the name of the class.